docs/Lab2/README.md

```diff
@@ -1,4 +1,4 @@
-# Lab 2. I need to change but want none of the hassle
+# Lab 2. Make changes with Helm
```

Comment:
Would have been nice to keep the old title .. less formal?

Reply:
As these would be used as page headers and in a table of contents i thought more formal titles were appropriate. see https://ibm.github.io/openshift101/exercise-01/ as an example on the left nav. i was worried that in a nav the less formal titles wouldn't make sense - particularly "I like sharing". :)

If this is a sticking point for you I'm happy to reverse the changes, just wanted to explain why I did that.

Reply:
Sounds reasonable.